src/semidiscretization/semidiscretization_coupled.jl

```diff
@@ -332,7 +349,7 @@ end
 ################################################################################
 
 """
-    BoundaryConditionCoupled(other_semi_index, indices, uEltype)
+    BoundaryConditionCoupled(other_semi_index, indices, uEltype, coupling_converter)
 
 Boundary condition to glue two meshes together. Solution values at the boundary
 of another mesh will be used as boundary values. This requires the use
```

Comment:
Just a question on the strategy here. So the two meshes have a true interface (i.e. this is not a Chimera mesh) and the coupling between the two domains is done via a numerical flux with (possible) mortar functionality to properly pass information back and forth, right?

Reply:
The coupling part should have been part of a different PR. I removed that now.
The coupling is done via the exchange of boundary values. So far only matching meshes can be coupled, as no interpolation is implemented.